tests/relay_integration/lang/javascript/test_plugin.py

```diff
@@ -1655,3 +1562,542 @@ def test_html_file_with_query_param_ending_with_js_extension(self):
         event = self.post_and_retrieve_event(data)
 
         assert "errors" not in event.data
+
+    def test_expansion_with_debug_id(self):
+        project = self.project
+        release = Release.objects.create(organization_id=project.organization_id, version="abc")
+        release.add_project(project)
+        debug_id = "c941d872-af1f-4f0c-a7ff-ad3d295fe153"
+
+        compressed = BytesIO()
+        with zipfile.ZipFile(compressed, mode="w") as zip_file:
+            zip_file.writestr("files/_/_/file.min.js", load_fixture("file.min.js"))
+            zip_file.writestr("files/_/_/file1.js", load_fixture("file1.js"))
+            zip_file.writestr("files/_/_/file2.js", load_fixture("file2.js"))
+            zip_file.writestr("files/_/_/empty.js", load_fixture("empty.js"))
+            zip_file.writestr(
+                "files/_/_/file.wc.sourcemap.js", load_fixture("file.wc.sourcemap.js")
+            )
+
+            zip_file.writestr(
+                "manifest.json",
+                json.dumps(
+                    {
+                        "org": self.organization.slug,
+                        "release": release.version,
+                        "files": {
+                            "files/_/_/file.min.js": {
+                                "url": "~/file.min.js",
+                                "type": "minified_source",
+                                "headers": {
+                                    "content-type": "application/json",
```

Comment:
its rather `application/javascript` :-)
Do we even need/want the `content-type` in there? I don’t think we ever use it.